scripts_ksql/setup.sh

```diff
@@ -7,12 +7,12 @@ fi
 
 DOCKER_MEMORY=$(docker system info | grep Memory | grep -o "[0-9\.]\+")
 if (( $(echo "$DOCKER_MEMORY 7.0" | awk '{print ($1 < $2)}') )); then
```

Comment:
This needs to change to reflect 12GB (if we stick with 12GB)

Reply:
looks like we don't need 12GB now that we've fixed some other configs, I'll remove this